export_tokenizer.py

```diff
@@ -0,0 +1,43 @@
+from sentencepiece import SentencePieceProcessor
+
+sp_model = SentencePieceProcessor(model_file="tokenizer.model")
+
+vocab_size = sp_model.vocab_size()
+bos_id = sp_model.bos_id()
+eos_id = sp_model.eos_id()
+pad_id = sp_model.pad_id()
+print(vocab_size, bos_id, eos_id, pad_id)
+
+# for i in range(vocab_size): print(i, repr(sp_model.id_to_piece(i)))
+
+token_blob = []
+offsets = []
+offset = 0
+for i in range(vocab_size):
+    t = sp_model.id_to_piece(i)
+    if i == bos_id:
+        t = '\n<s>\n'
+    elif i == eos_id:
+        t = '\n</s>\n'
+    elif len(t) == 6 and t.startswith('<0x') and t.endswith('>'):
```

Comment:
??? some comments around here could be nice. i haven't dug into sp too much but this looks odd